.eslintignore

```diff
@@ -6,3 +6,4 @@ installer
 node_modules
 js/preload/_*.js
 tests/**
+js/notify/*.html
```

Comment:
i'm not seeing lint errors.  
npm run lint looks fine.
also .html shouldn't be run through linter, just .js files.